beakerx/js/src/plot/plotScope.js

```diff
@@ -2417,7 +2417,7 @@ define([
       self.model.model = data;
     }
 
-    // self.element = $('div#'+self.wrapperId+' .dtcontainer');
+    self.element = $('div#'+self.wrapperId+' .dtcontainer');
```

Comment:
@scottdraves it is not ok, this line should be removed.
@fishera123 you can change order of method execution in Plot.js (64,65 lines) file
set the element 
`currentScope.setElement(tmplElement.children('.dtcontainer'));`
before setting a model
`currentScope.setModelData(data);`